server/app/views/v1/host_nodes/token.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.id @node.node_id
```

Comment:
`@node.to_path` is better... but it's a bit confusing to return the same id as in node resource. Maybe it's fine here because token is "embedded" resource for a node.